src/main/java/model/CardGenerator.java

```diff
@@ -0,0 +1,46 @@
+package model;
+
+import java.util.Random;
+
+public class CardGenerator {
```

Comment:
I am not sure that should be kept in the `model` package.

Reply:
Moved it to default package